routes/web.php

```diff
@@ -65,38 +85,38 @@
-        Route::get('/imports/{import}/complete', 'ImportController@getComplete')->name('complete');
-        Route::post('/imports/{import}/complete', 'ImportController@postComplete');
-        Route::delete('/imports/{import}', 'ImportController@destroy');
+        Route::get('/imports', [ImportController::class, 'index'])->name('index');
+        Route::get('/imports/create', [ImportController::class, 'create'])->name('create');
+        Route::post('/imports', [ImportController::class, 'store'])->name('store');
+        Route::get('/imports/{import}/prepare', [ImportController::class, 'getPrepare'])->name('prepare');
+        Route::post('/imports/{import}/prepare', [ImportController::class, 'postPrepare']);
+        Route::get('/imports/{import}/complete', [ImportController::class, 'getComplete'])->name('complete');
+        Route::post('/imports/{import}/complete', [ImportController::class, 'postComplete']);
+        Route::delete('/imports/{import}', [ImportController::class, 'destroy']);
     });
 
     Route::name('activities.')->group(function () {
-        Route::get('/activities', 'ActivityController@index')->name('index');
+        Route::get('/activities', [ActivityController::class, 'index'])->name('index');
     });
 
     Route::name('settings.')->group(function () {
-        Route::get('/settings', 'SettingsController@getIndex')->name('index');
-        Route::post('/settings', 'SettingsController@postIndex');
-        Route::get('/settings/profile', 'SettingsController@getProfile')->name('profile');
-        Route::get('/settings/account', 'SettingsController@getAccount')->name('account');
-        Route::get('/settings/preferences', 'SettingsController@getPreferences')->name('preferences');
-        Route::get('/settings/spaces', 'SettingsController@getSpaces')->name('spaces.index');
+        Route::get('/settings', [SettingsController::class, 'getIndex'])->name('index');
+        Route::post('/settings', [SettingsController::class, 'postIndex']);
+        Route::get('/settings/profile', [SettingsController::class, 'getProfile'])->name('profile');
+        Route::get('/settings/account', [SettingsController::class, 'getAccount'])->name('account');
+        Route::get('/settings/preferences', [SettingsController::class, 'getPreferences'])->name('preferences');
+        Route::get('/settings/spaces', [SettingsController::class, 'getSpaces'])->name('spaces.index');
     });
 
-    Route::get('/spaces/{id}', 'SpaceController');
+    Route::get('/spaces/{id}', SpaceController::class);
 
     Route::name('ideas.')->group(function () {
-        Route::get('/ideas/create', 'IdeaController@create')->name('create');
-        Route::post('/ideas', 'IdeaController@store');
+        Route::get('/ideas/create', [IdeaController::class, 'create'])->name('create');
+        Route::post('/ideas', [IdeaController::class, 'store']);
     });
 
     Route::get('/translations', function () {
```

Comment:
TIP Try to avoid closures, this breaks laravel route caching.

Reply:
#191 🙂